plugins/inputs/radius/radius.go

```diff
@@ -0,0 +1,104 @@
+package radius
+
+import (
+	"context"
+	_ "embed"
+	"net"
+	"sync"
+	"time"
+
+	"layeh.com/radius"
+	"layeh.com/radius/rfc2865"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/config"
+	"github.com/influxdata/telegraf/plugins/inputs"
+)
+
+type Radius struct {
+	Servers         []string
+	Username        string
+	Password        string
+	Secret          string
+	ResponseTimeout config.Duration
+	Log             telegraf.Logger
+}
+
+//go:embed sample.conf
+var sampleConfig string
+
+func (n *Radius) SampleConfig() string {
+	return sampleConfig
+}
+
+func (n *Radius) Description() string {
+	return "Test Radius authentication"
+}
+
```

Comment:
This function can be removed as the description has moved to `sample.conf` (comment above the actual TOML).